packages/react/src/hooks/store.hook.ts

```diff
@@ -0,0 +1,35 @@
+export interface StoreInterface {
+  authenticationToken: {
+    get: () => string | undefined;
+    set: (token: string) => void;
+    remove: () => void;
+  };
+}
+
+enum StoreKey {
+  AUTH_TOKEN = "authenticationToken",
+}
+
+export function useStore(): StoreInterface {
+  const { localStorage } = window;
+
+  function set(key: StoreKey, value: string) {
+    localStorage.setItem(key, value);
+  }
+
+  function get(key: StoreKey): string | undefined {
+    return localStorage.getItem(key) ?? undefined;
+  }
+
+  function remove(key: StoreKey) {
+    localStorage.removeItem(key);
+  }
```

Comment:
I think the storage API should be provided from outside so that the package can also be used in React Native applications.

Reply:
yes I have thought about this too, but decided against it in first iteration. We can do a ticket about it, because it will be a greater change, IMHO

Reply:
Added a ticket